--- origsrc/freetds-0.91/configure.ac	2011-08-17 20:55:22.000000000 -0500
+++ src/freetds-0.91/configure.ac	2014-01-01 14:47:19.084200700 -0600
@@ -153,13 +153,7 @@ tds_mingw=no
 case $host in
 *-*-mingw*)
 	tds_mingw=yes
-	if test "$host_cpu" = "x86_64"; then
-		LIBS="-lws2_32"
-	elif test -r /usr/lib/w32api/libws2_32.a; then
-		LIBS="-L/usr/lib/w32api -lws2_32"
-	else
-		LIBS="-lws2_32"
-	fi
+	LIBS="-lws2_32"
 	krb5_libs=
 	AM_CONDITIONAL(MINGW32, true)
 	;;
--- origsrc/freetds-0.91/src/ctlib/Makefile.am	2014-01-01 19:00:05.646187400 -0600
+++ src/freetds-0.91/src/ctlib/Makefile.am	2014-01-01 13:52:13.000000000 -0600
@@ -18,5 +18,5 @@ SYMBOLS		=	-export-symbols-regex '^(cs_|
 endif
 libct_la_LDFLAGS=	-version-info 4:0:0 $(SYMBOLS) $(FREETDS_SYMBOLIC)
 
-libct_la_LIBADD	=	../tds/libtds.la ../replacements/libreplacements.la $(NETWORK_LIBS) $(LIBICONV) $(FREETDS_LIBGCC)
+libct_la_LIBADD	=	../tds/libtds.la ../replacements/libreplacements.la $(NETWORK_LIBS) $(LTLIBICONV) $(FREETDS_LIBGCC)
 
--- origsrc/freetds-0.91/src/dblib/Makefile.am	2014-01-01 19:00:05.650688000 -0600
+++ src/freetds-0.91/src/dblib/Makefile.am	2014-01-01 13:52:13.000000000 -0600
@@ -20,5 +20,5 @@ else
 SYMBOLS		=	-export-symbols-regex '^(db|bcp_|tdsdump_open|tdsdbopen|.*_xact|close_commit|open_commit|.?asprintf).*'
 endif
 libsybdb_la_LDFLAGS=	-version-info 5:0:0 $(SYMBOLS) $(FREETDS_SYMBOLIC)
-libsybdb_la_LIBADD=	../tds/libtds.la ../replacements/libreplacements.la $(NETWORK_LIBS) $(LIBICONV) $(FREETDS_LIBGCC)
+libsybdb_la_LIBADD=	../tds/libtds.la ../replacements/libreplacements.la $(NETWORK_LIBS) $(LTLIBICONV) $(FREETDS_LIBGCC)
 
--- origsrc/freetds-0.91/src/odbc/Makefile.am	2014-01-01 19:00:05.655188300 -0600
+++ src/freetds-0.91/src/odbc/Makefile.am	2014-01-01 13:52:13.000000000 -0600
@@ -20,7 +20,7 @@ libtdsodbc_la_LDFLAGS = -Wl,--kill-at -W
 .rc.res:
 	$(RC) -i $< -I $(top_builddir)/win32 --input-format=rc -o $@ -O coff
 else
-libtdsodbc_la_LIBADD=	../tds/libtds.la ../replacements/libreplacements.la $(ODBCINSTLIB) $(NETWORK_LIBS) $(LIBICONV) $(FREETDS_LIBGCC)
+libtdsodbc_la_LIBADD=	../tds/libtds.la ../replacements/libreplacements.la $(ODBCINSTLIB) $(NETWORK_LIBS) $(LTLIBICONV) $(FREETDS_LIBGCC)
 if MACOSX
 SYMBOLS		=
 else
--- origsrc/freetds-0.91/src/server/Makefile.am	2014-01-01 19:00:05.662188700 -0600
+++ src/freetds-0.91/src/server/Makefile.am	2014-01-01 13:52:13.000000000 -0600
@@ -2,7 +2,7 @@
 AM_CPPFLAGS	=	-I$(top_srcdir)/include
 noinst_LTLIBRARIES	=	libtdssrv.la
 libtdssrv_la_SOURCES=	query.c server.c login.c
-libtdssrv_la_LIBADD =	../tds/libtds.la ../replacements/libreplacements.la $(NETWORK_LIBS) $(LIBICONV) $(FREETDS_LIBGCC)
+libtdssrv_la_LIBADD =	../tds/libtds.la ../replacements/libreplacements.la $(NETWORK_LIBS) $(LTLIBICONV) $(FREETDS_LIBGCC)
 noinst_PROGRAMS	= tdssrv
 tdssrv_LDADD	= libtdssrv.la $(LIBICONV) $(NETWORK_LIBS)
 tdssrv_SOURCES	= unittest.c
--- origsrc/freetds-0.91/src/tds/Makefile.am	2014-01-01 19:00:05.668189000 -0600
+++ src/freetds-0.91/src/tds/Makefile.am	2014-01-01 13:52:13.000000000 -0600
@@ -19,7 +19,7 @@ libtds_la_SOURCES=	mem.c token.c util.c
 	bulk.c win_mutex.c \
 	$(AUTH_FILES)
 libtds_la_LDFLAGS=
-libtds_la_LIBADD=
+libtds_la_LIBADD= $(LTLIBICONV)
 
 noinst_HEADERS		= tds_willconvert.h encodings.h num_limits.h tds_types.h
 EXTRA_DIST		= tds_willconvert.h encodings.h num_limits.h tds_types.h \
